great_expectations/expectations/core/expect_column_values_to_be_between.py

```diff
@@ -352,7 +352,7 @@ def _atomic_prescriptive_template(
             template_str = f"{conditional_template_str}, then {template_str}"
             params_with_json_schema.update(conditional_params)
 
-        return (template_str, params_with_json_schema, styling)
+        return template_str, params_with_json_schema, None, styling
```

Comment:
These `None`s are placeholders for `meta_notes` until these `_atomic_prescriptive_template` methods are refactored into `_prescriptive_template` methods that use `RendererConfiguration`.

Reply:
The superclass method has been refactored in a generic way, but it can't be removed until all sub-class methods are first removed because it is not always overriden.